packages/core/src/connectors/utilities/loader.ts

```diff
@@ -0,0 +1,68 @@
+import path from 'path';
+
+import type { AllConnector, CreateConnector } from '@logto/connector-kit';
+import connectorKitMeta from '@logto/connector-kit/package.json' assert { type: 'json' };
+import { satisfies } from 'semver';
+
+const connectorKit = '@logto/connector-kit';
+const { version: currentVersion } = connectorKitMeta;
+
+const isKeyInObject = <Key extends string>(
+  object: unknown,
+  key: Key
+  // eslint-disable-next-line @typescript-eslint/ban-types
+): object is object & Record<Key, unknown> =>
```

Comment:
nit: Would it be OK to just specify `object is Record<key, unknown>`?

Reply:
then you cannot access other keys

Reply:
yeah, you're right... sorry